source/NVDAObjects/behaviors.py

```diff
@@ -400,12 +400,16 @@ class KeyboardHandlerBasedTypedCharSupport(Terminal):
 	characters close to the caret, or injecting in-process with NVDAHelper.
 	This class relies on the toUnicodeEx Windows function, and in particular
 	the flag to preserve keyboard state available in Windows 10 1607
-	and later."""
+	and later.
+	This class also can hold typed characters in a queue and only dispatch once
+	the screen updates. This is useful for suppression of passwords, etc."""
 	#: Whether this object quickly and reliably sends textChange events
 	#: when its contents update.
 	#: Timely and reliable textChange events are required
 	#: to support password suppression.
 	_supportsTextChange = True
+	#: Whether to use ToUnicodeEx for typed character events.

```

Comment:
What's the benefit of this class when this is False?

Reply:
Catches typed characters that aren't written to the screen (such as passwords), handles tab completion, etc.

Reply:
AH I see, so it gets the keybord input by other means.